javers-persistence-mongo/src/main/java/org/javers/repository/mongo/MongoRepository.java

```diff
@@ -28,6 +30,11 @@
     public static final String SNAPSHOTS = "jv_snapshots";
     public static final String COMMIT_ID = "commitMetadata.id";
```

Comment:
mongo źle sortuje po id, np wg mongo 2.0 > 14.0, trzeba to fixnąć pisałem na hipchat